src/OpenLoco/Audio/VehicleChannel.cpp

```diff
@@ -1,18 +1,94 @@
 #include "VehicleChannel.h"
 #include "../Entities/EntityManager.h"
 #include "../Interop/Interop.hpp"
+#include "../Map/TileManager.h"
+#include "../Ui/WindowManager.h"
 #include "../Vehicles/Vehicle.h"
 
 using namespace OpenLoco;
 using namespace OpenLoco::Audio;
 using namespace OpenLoco::Interop;
 
+uint8_t getZoomVolumeModifier(uint8_t zoom)
+{
+    switch (zoom)
+    {
+        case 0:
+            return 0;
+        case 1:
+            return 35;
+        default:
+            return 70;
+    }
```

Comment:
You could do this without branching.
```suggestion
    return std::min(2, zoom) * 35;
```